rfcs/0001-pip-install-rearchitecture.md

```diff
@@ -0,0 +1,49 @@
+# Pip Install Rearchitecture
+## Proposal
+The existing pip buildpack should be rewritten and restructured to only provide the pip dependency. The pip install logic should be factored out into it's own buildpack.
+
+## Motivation
+In keeping with the overarching [Python Buildpack Rearchitecture RFC](https://github.com/paketo-community/pip/blob/main/rfcs/0001-pip-rearchitecture.md), the Pip Install Buildpack should perform one task, which is installing from requirements files. This is part of the effort in Paketo Buildpacks to reduce the responsibilities of each buildpack to make them easier to understand and maintain.
+
+## Implementation
+### API
+- pip-install
+  - `requires`: `cpython` and `pip` during build 
+  - `provides`: `site-packages`
+
+### Detect
+The pip-install buildpack should only detect if there is a `requirements.txt` file at the root of the app.
+
+### Build
+#### Configuration
+There will be two layers, packages layer and cache layer. 
+The packages layer will contain the result of the pip install command.
+The cache layer will hold the pip [cache](https://pip.pypa.io/en/stable/reference/pip_install/#caching).
+
+During the build process, the resulting build command will be:
+```bash
+python -m pip install
```

Comment:
Does this run the `pip` module that was installed by the `pip` buildpack, or the one that is included in Python by default?

Reply:
I guess I'm asking why we have to run `python -m pip` instead of just running `pip`. Is there an explanation for that difference?

Reply:
^ `pip` from the Pip buildpack should be on the path by the time this buildpack runs

Reply:
That was on oversight. fixed